server/src/constants.ts

```diff
@@ -17,6 +18,8 @@ export const ADDED_IN_PREFIX = 'This property was added in ';
 export const JOBS_ASSET_PAGINATION_SIZE = 1000;
 export const JOBS_LIBRARY_PAGINATION_SIZE = 10_000;
 
+export const PLUGIN_JWT_SECRET = randomBytes(32).toString('base64');
```

Comment:
Is there any chance of this causing issues with restarts or multiple instances? I don't think so right?

Reply:
As in, a process trying to verify a JWT against another process's secret.